features/dynamic-bundle-discounts.md

```diff
@@ -0,0 +1,210 @@
+# Dynamic Bundle discounts
+
+- [Dynamic Bundle discounts](#dynamic-bundle-discounts)
+  - [Introduction](#introduction)
+  - [Setup](#setup)
+    - [Nosto Dynamic Bundle template](#nosto-dynamic-bundle-template)
+    - [Shopify Line Item script](#shopify-line-item-script)
+    - [Known Issues](#known-issues)
+
+## Introduction
+This documentation explains the process of setting up `Nosto - Dynamic Bundles` for automatic discount, when a bundle is added to the cart on a Shopify store. As per the existing setup, the end-to-end process has to be done manually. This release improves this by reducing manual intervention, to an extent, in setting up the store for bundle discounts.
```

Comment:
```suggestion
This documentation explains the process of setting up Dynamic Bundles for automatic discount, when a bundle is added to the cart on a Shopify store.
```

The \` parenthesis is usually used for code snippets. It's better to not use them in other cases e.g. `Nosto - Dynamic Bundles` is less confusing as: Nosto - Dynamic Bundles

This text is a bit confusing to read. I'd leave it out to make the introduction more concise:
> As per the existing setup, the end-to-end process has to be done manually. This release improves this by reducing manual intervention, to an extent, in setting up the store for bundle discounts.

Reply:
I have remove Backtick usages and using bold and italics to highlight areas